Solutions/Zscaler Private Access (ZPA)/Analytic Rules/ZscalerZPAConnectionsOutsideOperationalHours.yaml

```diff
@@ -0,0 +1,32 @@
+id: 2859ad22-46c8-4cc7-ad7b-80ce0cba0af3
+name: Zscaler - ZPA connections outside operational hours
+description: |
+  'Detects ZAP connections outside operational hours.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: ZscalerPrivateAccess
+    dataTypes:
+      - ZPAEvent
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - InitialAccess
+relevantTechniques:
+  - T1190
+  - T1133
+query: |
+  ZPAEvent 
+  | where DvcAction == 'open'
+  | where datetime_part("Hour", TimeGenerated) between (datetime_part("Hour", todatetime('19:01:00.7654321')) .. datetime_part("Hour", todatetime('08:01:00.7654321')))

```

Comment:
Since time stamps are UTC, this detection, as is, does not make sense globally.

Reply:
fixed